aws-observability/json/Alerts-App.json

```diff
@@ -2994,7 +2994,7 @@
     },
     {
       "name": "Amazon RDS PostgreSQL - High Errors",
-      "description": "This alert fires when we detect Postgres logs show high rate of error/fatal logs",
+      "description": "This alert fires when we detect high rate (>10) of error/fatal logs in Postgres logs over a 5 minutes time period",
```

Comment:
Please cross check if it should be rate or number?

Reply:
As log, it should be number